### ISSUE TITLE
seems to be missing "n" (next message in thread) support

### ISSUE THREAD

**Issue by trak3r** [nompute/GMailinator#19]
if i can figure it out i'll submit a pull request, but i'm not an obj-c developer so i'm just throwing spaghetti at the wall so far.

**Comment by nompute** [comment]
Are you talking about keyboard navigation within the message (right-most) view?  The next/prev message methods in Mail.app broke in Mavericks (even the drop-down menu options don't work), so it's possible the method is just broken.  This is an unfortunately common occurrence with Mail.

**Comment by trak3r** [comment]
> Are you talking about keyboard navigation within the message (right-most) view? 

Yes. I just discovered through trial and error that OPTION-UPARROW and OPTION-DOWNARROW are the default keys. Would like those to be replaced with "n" and "p" as Gmail does.
